src/index.ts

```diff
@@ -1,5 +1,6 @@
 export {
   parse,
+  print,
```

Comment:
nit: if we do go for it, I'd instead add a re-export here explicitly:

```js
export { print } from '@0no-co/graphql.web';
```

The indirection of import+export causes some older versions of some bundlers some trouble when treeshaking.

Reply:
Just pushed that instead